### ISSUE TITLE
ClassNotFoundException when downloading v2.4.2 from Maven Central

### ISSUE THREAD

**Issue by mmoayyed** [spring-projects/spring-session-data-mongodb#183]
### Expected behavior

Successfully download the relevant JAR from Maven Central for version 2.4.2.

### Actual behavior

Attempting to download version 2.4.2 from Central produces ClassNotFoundException errors.
https://search.maven.org/artifact/org.springframework.session/spring-session-data-mongodb/2.4.2/jar

```
error: package org.springframework.session.data.mongo.config.annotation.web.http does not exist
import org.springframework.session.data.mongo.config.annotation.web.http.EnableMongoHttpSession;
```

### Steps to reproduce

Declare the dependency and attempt to let Maven or Gradle resolve it.

The JAR that is published on central does not contain any relevant classes, etc if you download and examine it manually.

![image](https://user-images.githubusercontent.com/1205228/105459477-8bcdae00-5c9f-11eb-9658-5b0ae112b6e7.png)

### JVM version (e.g. `java -version`)

OpenJDK 11.0.9, Gradle 6.8

### OS version (e.g. `uname -a`)

Darwin MacBook-Pro 20.2.0 Darwin Kernel Version 20.2.0

**Comment by gregturn** [comment]
Can you show me your build file?

**Comment by mmoayyed** [comment]
- https://github.com/apereo/cas/blob/master/support/cas-server-support-session-mongo/build.gradle#L5
- https://github.com/apereo/cas/blob/master/gradle/dependencies.gradle#L3200

Don't think it's a build issue. You can see in my screenshot above that the JAR is empty, and this also works with 2.4.1.

**Comment by gregturn** [comment]
Wow. Confirmed.

I spot checked, and the latest (2.5.0-M2) is intact so digging in to see what went wrong in the release process.

**Comment by gregturn** [comment]
Just rebuilt `2.4.2` locally, and everything was intact.

Attempting to push another release, but this is delaying releasing => https://twitter.com/sonatype_ops/status/1352625400480817153

**Comment by gregturn** [comment]
Resolved with version 2.4.3.

**Comment by gregturn** [comment]
Closed via https://github.com/spring-projects/spring-session-data-mongodb/commit/2fa62693d79ac9275a2ddb8030ce6d12a0d75162.
